lib/chat/chat.dart

```diff
@@ -1,18 +1,72 @@
+import 'package:google_fonts/google_fonts.dart';
 import 'package:flutter/material.dart';
+import './new_matches.dart';
+import './messages.dart';
 
-// TODO: Update this placeholder widget
 class Chat extends StatelessWidget {
   const Chat({Key key}) : super(key: key);
 
   @override
   Widget build(BuildContext context) {
-    return Scaffold(
-      backgroundColor: Colors.black,
-      body: Container(
-        child: Center(
-          child: Text("Matches", style: TextStyle(color: Colors.white, fontSize: 30.0))
-        )
-      )
+    return Stack(
+      children: <Widget>[
+        Scaffold(
+          backgroundColor: Colors.transparent,
+          body: Container(
+            height: MediaQuery.of(context).size.height,
+            margin: const EdgeInsets.all(0),
+            child: Center(
+              child: Column(
+                children: buildChatView()
+              )
+            )
+          )
+        ),
+      ]
     );
   }
+}
+
+buildHeader() {
+  return Container (
+    margin: const EdgeInsets.all(30.0),
```

Comment:
The header gets cut off on some screen devices, but we'll handle this later in another PR.